.backportrc.json

```diff
@@ -11,5 +11,5 @@
     "^v9.3.0$": "main",
     "^v(\\d+).(\\d+)(.\\d+)*$": "$1.$2"
   },
-  "upstream": "elastic/connectors"
+  "upstream": "elastic/app/connectors_service/connectors"
```

Comment:
Hmmm I believe `upstream` is the git repo path, not path to a file?

Reply:
Ah you're right, let me revert. I didn't realise it was the deprecated version of `repoOwner` & `repoName`.